UT4MasterServer.Web/src/pages/Servers.vue

```diff
@@ -5,8 +5,9 @@
         <div class="list-group">
           <a class="list-group-item list-group-item-action active">Hubs</a>
           <Hub :hub="hub" v-for="hub in ServerStore.hubs" :selected-hub="selectedHub" @click="viewHub(hub)" />
+          <Hub v-if="!ServerStore.hubs.length" />
         </div>
-        <!-- TODO: Should also have Servers listed here in addition to Hubs, but I'm not yet sure how to differentiate -->
+        <!-- TODO: Should(?) also have Servers listed here in addition to Hubs, but I'm not yet sure how to differentiate -->
```

Comment:
There should be a way to switch between listing hubs and servers eventually. same as in the game. (maybe additionally even qp matches once we get there)